components/authn-service/tokens/util/tokens.go

```diff
@@ -21,7 +21,7 @@ func MinimumTokenLength() int {
 func IsValidLegacyToken(token string) error {
 	if len(token) < MinimumLegacyTokenLength {
 		return fmt.Errorf(
-			"legacy token %q is invalid; it is length %d but must be at least %d characters",
+			"legacy token %q is invalid; it has length %d but must be at least %d characters",
```

Comment:
🤓

Reply:
I mean if we're fixing it maybe "it has X characters should but be at least Y characters" but also w/e